src/pages/task/Metadata.tsx

```diff
@@ -183,6 +186,12 @@ export const Metadata: React.FC<Props> = ({ loading, data, error, taskId }) => {
           ))}
         </span>
       ) : null}
+      {oomTracker && oomTracker.detected && (
+        <P2>
```

Comment:
Check with design but maybe this should look different to emphasis that this event happened

Reply:
I reached out to design. Thanks!